### DISCUSSION TITLE
I have 8 Apps on Google Play with old certificates. Cannot create APK with Harman 33.1.217

### DISCUSSION THREAD

**Discussion by Menard1965** [airsdk/Adobe-Runtime-Support#628] [Q&A]
## Problem Description

Please help!
I want to update my Apps on Google Paly i used a 1024 bit certificate 5 Years ago. Since Animate 2020 i can not use this certificate to create APK java is not  allowing that. So i copied the JRE from Adobe flash cc.2015 to the common files Adobe Animate 2020. replaced the current JRE.  This was possible until 2 month ago.  

Now  with the new SDK 3.1.1.217 from Harman i grt the following error

**Wrong arguments. -arch must be followed by (armv7 | x86)**

I use Adobe Animate latest version (blue)

Plerase help. There is an urgent Update that i have to upload to google Play.

[https://www.blue-2.at/VST/Error1.JPG](https://www.blue-2.at/VST/Error1.JPG)

Thanks Michael

**Comment by ajwfrost** [comment]
If you're seeing that error then it looks like it's not calling the right version of the AIR SDK. What are you selecting for the CPU architecture in Animate 2020?
It may be worth removing and re-adding the AIR SDK v33.1.1.217 in Animate and make sure that the project's settings are pointing to use this new version?

**Comment by Menard1965** [comment]
Thanks!

 

CPU architecture is CPU

I will test it removing the SDK and add it from Scratch. 

Thanks a lot.

 

 

Von: Andrew Frost <notifications@github.com> 
Gesendet: Montag, 21. September 2020 07:53
An: Gamua/Adobe-Runtime-Support <Adobe-Runtime-Support@noreply.github.com>
Cc: Menard1965 <menard@blue-2.at>; Author <author@noreply.github.com>
Betreff: Re: [Gamua/Adobe-Runtime-Support] I have 8 Apps on Google Play with old certificates. Cannot create APK with Harman 33.1.217 (#472)

 

If you're seeing that error then it looks like it's not calling the right version of the AIR SDK. What are you selecting for the CPU architecture in Animate 2020?
It may be worth removing and re-adding the AIR SDK v33.1.1.217 in Animate and make sure that the project's settings are pointing to use this new version?
